tests/src/test/scala/whisk/core/cli/test/ApiGwRestBasicTests.scala

```diff
@@ -899,4 +900,37 @@ abstract class ApiGwRestBasicTests extends BaseApiGwTests {
       apiDelete(basepathOrApiName = testbasepath, expectedExitCode = DONTCARE_EXIT)
     }
   }
+
+  it should "verify get API name that uses custom package" in {
+    val testName = "CLI_APIGWTEST25"
+    val testbasepath = "/" + testName + "_bp"
+    val testrelpath = "/path"
+    val testnewrelpath = "/path_new"
+    val testurlop = "get"
+    val testapiname = testName + " API Name"
+    val packageName = "pkg_" + UUID.randomUUID()
+    val actionName = packageName + "/" + testName + "_action"
+    try {
+      wsk.pkg.create(packageName).stdout should include regex (s""""name":\\s*"$packageName"""")
```

Comment:
this could use the withAssetCleaner helper which will delete the action on its own at the end of the test.